source/brailleDisplayDrivers/seikantk.py

```diff
@@ -97,7 +97,7 @@ def isSeikaBluetoothDeviceMatch(match: DeviceMatch) -> bool:
 
 
 class BrailleDisplayDriver(braille.BrailleDisplayDriver):
-	_dev: hwIo.IoBase
+	_dev: Optional[hwIo.IoBase] = None
```

Comment:
Should this actually be made an instance variable?

Reply:
this has been changed in a146544